README.md

```diff
@@ -1,4 +1,4 @@
-# Sample Plugin [![Build Status](https://travis-ci.org/mattermost/mattermost-plugin-sample.svg?branch=master)](https://travis-ci.org/mattermost/mattermost-plugin-sample)
+# Sample Plugin [![CircleCI](https://circleci.com/gh/mattermost/mattermost-plugin-sample.svg?style=svg)](https://circleci.com/gh/mattermost/mattermost-plugin-sample)
```

Comment:
Nit: What do you think about using https://shields.io/ instead for the unified style of the badge?

Reply:
we can use nothing against